src/houghcircle.jl

```diff
@@ -0,0 +1,134 @@
+"""
+```
+circle_centers, circle_radius = hough_circle_gradient(img, scale, min_dist, canny_thres, vote_thres, min_radius, max_radius)
+```
+Returns two vectors, corresponding to circle centers and radius.
+
+The circles are generated using a hough transform variant in which a non-zero point only votes for circle
+centers perpendicular to the local gradient.
+
+Parameters:
+    img          = image to detect circles in
+    scale        = relative accumulator resolution factor
+    min_dist     = minimum distance between detected circle centers
+    canny_thres  = upper threshold for canny, lower threshold=upper threshold/4
+    vote_thres   = accumulator threshold for circle detection
+    min_radius   = minimum circle radius
+    max_radius   = maximum circle radius
```

Comment:
You could cut down on the number of parameters by using an argument `radii::AbstractVector` and have the user call this as `hough_circle_gradient(..., min_radius:max_radius)`.